modules/@apostrophecms/area/ui/apos/components/ApostropheAreaEditor.vue

```diff
@@ -47,74 +47,124 @@ export default {
       return Object.keys(this.options.widgets);
     }
   },
+  watch: {
+    next() {
+      if (!this.docId) {
+        this.$emit('changed', {
+          items: this.next
+        });
+      }
+    }
+  },
   methods: {
     async up(i) {
-      await apos.http.patch(`${apos.doc.action}/${this.docId}`, {
-        busy: true,
-        body: {
-          $move: {
-            [`@${this.id}.items`]: {
-              $item: this.next[i].widget._id,
-              $before: this.next[i - 1].widget._id
+      if (this.docId) {
```

Comment:
It works in the modal, but this button appears on the page, and when clicking it, there is this error: 

```js
[Vue warn]: Error in v-on handler (Promise/async): "[object XMLHttpRequest]"

found in

---> <ApostropheAreaEditor> at apostrophe/modules/@apostrophecms/area/ui/apos/components/ApostropheAreaEditor.vue
       <Root>
```

Maybe remove the button on the page if it is intended to be modified only in the modal

Reply:
Both the up and the down button work for me "on the page" as well, please clarify how to reproduce this issue with steps. Thanks!

Reply:
Tried a new project to be sure, still unable to reproduce an issue with the up or down button "on the page."